tensorhive/controllers/reservation.py

```diff
@@ -153,7 +150,8 @@ def delete(id: ReservationId) -> Tuple[Content, HttpStatusCode]:
         reservation_to_destroy = Reservation.get(id)
 
         # Must be privileged
-        assert __is_admin_or_reservation_owner(reservation_to_destroy), GENERAL['unprivileged']
+        assert (reservation_to_destroy.start > datetime.datetime.utcnow() and __is_reservation_owner(
+            reservation_to_destroy)) or __is_admin(), GENERAL['unprivileged']
```

Comment:
Maybe we should update the comment

Reply:
fixed in aa2b66e981dd77f605e16f49b763e22ffc341817